llvm/test/CodeGen/AMDGPU/sgpr-copy.ll

```diff
@@ -256,7 +256,7 @@ endif:                                            ; preds = %else, %if
 define amdgpu_kernel void @copy1(ptr addrspace(1) %out, ptr addrspace(1) %in0) {
 entry:
   %tmp = load float, ptr addrspace(1) %in0
-  %tmp1 = fcmp oeq float %tmp, 0.000000e+00
+  %tmp1 = fcmp one float %tmp, 0.000000e+00
```

Comment:
Without this change, a cmov pattern will be emitted, and `v_add` is missing.

Reply:
This change is used to avoid changing check lines :)

Reply:
Addition of `isImpliedCondFCmps` affects the IR is what you're saying I believe, in that case shouldn't the result be updated rather than changing the test itself?

Reply:
I am not familiar with the AMDGPU target. Updating check lines may change the intent of the original test.

Reply:
Yeah, I guess @arsenm can have a look at this.